app/src/main/java/ch/epfl/sweng/favors/database/storage/FirebaseStorageDispatcher.java

```diff
@@ -15,6 +19,9 @@
 public abstract class FirebaseStorageDispatcher {
 
     private static FirebaseStorageDispatcher storage = null;
+    public final static int GET_FROM_GALLERY = 66;
+    public final static int GET_FROM_CAMERA = 99;
```

Comment:
From nowhere :) They are just identifiers for the methods that load pictures from camera and gallery

Reply:
so, why not using an enum here too ?

Reply:
I think that we need to use integers because of the method signatures that we are overiding.
I found an example produced by google that is similar to what Andrea has done. https://github.com/GoogleCloudPlatform/cloud-vision/blob/master/android/CloudVision/app/src/main/java/com/google/sample/cloudvision/MainActivity.java